runtime/gc_check/CheckEngine.cpp

```diff
@@ -460,14 +462,14 @@ GC_CheckEngine::checkJ9Object(J9JavaVM *javaVM, J9Object* objectPtr, J9MM_Iterat
 		}
 	}
 
-#if defined(J9VM_ENV_DATA64)
```

Comment:
I think we should not remove this line. DataAddr helpers are defined for 64 bit only. I suspect this code can not be compiled on 32-bit platform